.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
what's this?

Reply:
Its a folder added automatically by husky when you run `husky install`

Reply:
fine then!